cli/pkg/orchestrator/orchestrator.go

```diff
@@ -69,17 +71,32 @@ func (s *Orchestrator) RunTasks(ctx context.Context, stack *stack_mgr.Stack, tas
 	launchType := s.happyConfig.TaskLaunchType()
 
 	tasks := []string{}
-	for _, taskOutput := range taskOutputs {
-		task, ok := stackOutputs[taskOutput]
-		if !ok {
-			continue
+	if taskArns, ok := stackOutputs["task_arns"]; ok {
```

Comment:
Original happy stack modules have individual outputs for every task arn; new universal modules refer to a map of `task_arns`. This change allows us to support both.